starter-code/js/index.js

```diff
@@ -49,4 +56,22 @@ window.onload = function(){
   for(var i = 0; i<deleteButtons.length ; i++){
     deleteButtons[i].onclick = deleteItem;
   }
-};
+};*/
+
+
+/*var button = document.getElementById("buttonCalculatPrice");
+button.onclick = function(){
+console.log(getPriceByProduct());
+};*/
+
+function totalPrice() {
+  var pricePerProducts= getPriceByProduct();
+  var subtotalPrice = document.getElementById('subtotalPrice');
+  subtotalPrice.innerHTML=pricePerProducts;
+}
+
+function deleteQuantity(){
```

Comment:
The delete button has to delete all the article instead of putting the quantity to 0.